.docker/scripts/create-test-data.sh

```diff
@@ -63,7 +113,7 @@ echo ${organisation_admin_update}
 echo "Updating publisher to have editor privileges in the department-of-health Organisation:"
 publisher_update=$( \
     curl -LsH "Authorization: ${API_KEY}" \
-    --data "id=department-of-health&username=publisher&role=editor" \
+    --data "id=department-of-health&username=editor&role=editor" \
```

Comment:
should this be in TEST_ORG_NAME also?

Reply:
I don't think we're using any tests where it matters, yet. This just keeps it in sync with ckanext-qgov.